pkg/netpol/connlist/connlist.go

```diff
@@ -119,6 +120,13 @@ func WithFocusWorkload(workload string) ConnlistAnalyzerOption {
 	}
 }
 
+// WithExposureAnalysis is a functional option to include exposure analysis
+func WithExposureAnalysis() ConnlistAnalyzerOption {
```

Comment:
I see that I misunderstood  the "can analysis be made while iterating pods and policies for connectivity analysis", thought we should return them in same results.
I will change the API's, will see how to use the existing connectivity-analysis funcs as computing allowed conns between two peers, without mixing the "results"

Reply:
we can have internally the computation for both at once, with "mixed results", but then separate the results before returning them from the main function...

Reply:
The separation can even be made in the internal `getConnectionsBetweenPeers()` function, where instead of appending all connections into one slice, you  append the separate "exposure connections" as exposure elements into a separate slice.

Reply:
handled in #296